### PR TITLE
[xcm] BridgeBlobDispatcher - prepend bridge instance discriminator.

### PR DESCRIPTION
**Pull request by bkontur**
https://github.com/paritytech/polkadot/pull/7438

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/3090958

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/3104315

File: parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_hub_rococo_config.rs [hunk 1]
```diff
@@ -46,6 +46,7 @@ parameter_types! {
 	pub const MaxUnconfirmedMessagesAtInboundLane: bp_messages::MessageNonce =
 		bp_bridge_hub_rococo::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
 	pub const BridgeHubWococoChainId: bp_runtime::ChainId = bp_runtime::BRIDGE_HUB_WOCOCO_CHAIN_ID;
+	pub BridgeWococoMessagesPalletInstance: Option<InteriorMultiLocation> = Some(X1(PalletInstance(<BridgeWococoMessages as PalletInfoAccess>::index() as u8)));
```
Comment (gavofyork) [inline parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_hub_rococo_config.rs:49]:
```suggestion
	pub BridgeWococoMessagesPalletInstance: InteriorMultiLocation = X1(PalletInstance(<BridgeWococoMessages as PalletInfoAccess>::index() as u8));
```

This should work, too.

File: parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_hub_wococo_config.rs [hunk 2]
```diff
@@ -47,6 +47,7 @@ parameter_types! {
 		bp_bridge_hub_wococo::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
 	pub const BridgeHubRococoChainId: bp_runtime::ChainId = bp_runtime::BRIDGE_HUB_ROCOCO_CHAIN_ID;
 	pub BridgeHubWococoUniversalLocation: InteriorMultiLocation = X2(GlobalConsensus(Wococo), Parachain(ParachainInfo::parachain_id().into()));
+	pub BridgeRococoMessagesPalletInstance: Option<InteriorMultiLocation> = Some(X1(PalletInstance(<BridgeRococoMessages as PalletInfoAccess>::index() as u8)));
```
Comment (gavofyork) [inline parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_hub_wococo_config.rs:50]: See above.

**Review by gavofyork — APPROVED** [review]

File: parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_hub_rococo_config.rs (hunk 1 above)
Comment (bkontur) [reply parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_hub_rococo_config.rs]: thank you, both changed

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/3108711

**Pull request merged**